File: Python/1154 - Ages.py
if __name__ == "__main__":
    done = False
    sum = 0
    counter = 0
    while done != True:
        x = int(input())
        if x < 0:
            done = True
        else:
            sum += x
            counter += 1
    avg = float((sum/counter))
    print("%.2f" % avg)

File: C++/1154 - Ages.cpp
#include <iostream>
#include <cstdio>
#include <vector>

std::vector<int> get_ages()
{
    std::vector<int> ages;
    int n;
    do
    {
        std::cin >> n;
        if (n < 0)
        {
            break;
        }
        ages.push_back(n);
    }
    while (n >= 0);
    return ages;
}

void average_age(std::vector <int> ages)
{
    int length, sum = 0;
    length = ages.size();
    for (int i = 0; i < length; i++)
    {
        sum += ages[i];
    }
    double avg = (sum/double(length));
    printf("%.2f\n",avg);
}

int main()
{
    std::vector<int> ages;
    ages = get_ages();
    average_age(ages);
    return 0;
}